### PR TITLE
Path corrections

### PR DESCRIPTION
**Pull request by davidbeckonline**
*Issue #, if available:*
The `deployment` folder was not considered correctly in the documentation and in the build-s3-dist.sh file.

*Description of changes:*
Updated the documentation and the build-s3-dist.sh file accordingly.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ DIST_OUTPUT_BUCKET=amcufa-$DATETIME-dist
 TEMPLATE_OUTPUT_BUCKET=amcufa-$DATETIME
 aws s3 mb s3://$DIST_OUTPUT_BUCKET-$REGION --region $REGION
 aws s3 mb s3://$TEMPLATE_OUTPUT_BUCKET --region $REGION
-./build-s3-dist.sh --template-bucket ${TEMPLATE_OUTPUT_BUCKET} --code-bucket ${DIST_OUTPUT_BUCKET} --solution-name amcufa --version ${VERSION} --region ${REGION} --profile ${PROFILE} | tee >( grep TEMPLATE >template )
+./deployment/build-s3-dist.sh --template-bucket ${TEMPLATE_OUTPUT_BUCKET} --code-bucket ${DIST_OUTPUT_BUCKET} --solution-name amcufa --version ${VERSION} --region ${REGION} --profile ${PROFILE} | tee >( grep TEMPLATE >template )
```
Comment (ianwow) [inline README.md:82]: Line 76 takes you into the deployment directory.

Comment (davidbeckonline) [reply README.md:82]: The `msg "$msg"` ?
This did not work for me in my tests.

Comment (ianwow) [reply README.md:82]: I don't understand. What error are you seeing and what command did you run to get that error? And is your PR resolving that error or is this a new issue?

Comment (davidbeckonline) [reply README.md:82]: I think this PULL request is obsolete. Sorry about that.

I made a mistake in step
`cd amazon-marketing-cloud-uploader-from-aws/deployment`
and just jumped to
`cd amazon-marketing-cloud-uploader-from-aws`
which explains the behavior I have been seeing.

Will run another test on this repository next week.

**Comment by davidbeckonline** [comment]
Obsolete.

**Pull request closed without merging**